controller/lib/core/sensors.cpp

```diff
@@ -21,10 +21,14 @@ Arduino Nano and the MPXV5004GP and MPXV7002DP pressure sensors.
 #include "hal.h"
 
 #include "blower.h"
+#include "math.h"
 #include "sensors.h"
 
-//@TODO: Replace floats with equivalent integer only operations to save program
-// space/time
+//@TODO: Potential Caution: Density of air slightly varies over temperature and
+// altitude - need mechanism to adjust based on delivery? Constant involving
+// density of air. Density assumed at 15 deg. Celsius and 1 atm of pressure.
+static const float ROOT_OF_TWO_OVER_DENSITY_OF_AIR =
+    sqrtf(2.0f / 1.225f); //[sqrt(meters^3/kg)]
```

Comment:
Can we cite where we got the density-of-air number from, or is this an obvious engineering number like pi?